core/create-app.tsx

```diff
@@ -145,6 +145,10 @@ export async function createApp(config: CreateMagicAppConfig) {
   }
   process.chdir(chosenProjectName);
 
+  if (fs.existsSync(`${cwd}/${chosenProjectName}/.env.example`)) {
+    fs.renameSync(`${cwd}/${chosenProjectName}/.env.example`, `${cwd}/${chosenProjectName}/.env`);
```

Comment:
Renaming the .env.example file to .env so that when the scaffold is run it wot throw env var not found error